app/models/episode.rb

```diff
@@ -115,16 +116,12 @@ def guid
     self[:guid]
   end
 
+  def explicit=(value)
+    super(Podcast::EXPLICIT_ALIASES[value] || value)
+  end
+
   def explicit_content
-    e = (explicit.blank? && podcast) ? podcast.explicit : explicit
-    e = e.to_s.downcase.strip
-    if ["clean", "no", "false", "f"].include?(e)
-      false
-    elsif ["explicit", "yes", "true", "t"].include?(e)
-      true
-    else
-      nil
-    end
+    (explicit || podcast&.explicit) == 'true'
```

Comment:
Oh, so this is interesting.  Dovetail (nodejs) actually uses this field, per #332.  [Dovetail actually concats](https://github.com/PRX/dovetail.prx.org/blob/master/lib/util/dovetail.js#L49) "explicit" or "clean" onto the episode's keywords, so they can be targeted in Adzerk.

The only change here is that `episode.explicitContent` will _never_ be null.  Checked with ad-ops, and that should be okay.

Reply:
makes sense to me - if anything, will make such targeting more useful.